### DISCUSSION TITLE
Fundamentos do blazor com .net 8 - Consigo usar o mysql?

### DISCUSSION THREAD

**Discussion by dalmiroe** [balta-io/forum#474] [Tira Dúvidas] [answered]
Boa noite. Gostaria de fazer este exemplo no mysql. 
Quando busco pelo pct vem este -  Pomelo.Entityframeworkcore.mysql.json.microsoft
Seria este mesmo?
Alguma obs?
Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
> Boa noite. Gostaria de fazer este exemplo no mysql. 
> 
> Quando busco pelo pct vem este -  Pomelo.Entityframeworkcore.mysql.json.microsoft
> 
> Seria este mesmo?
> 
> Alguma obs?
> 
> Obrigado

Boa tarde, @dalmiroe, como vai?

Não tenho muita experiência com MySQL, então não conheço este pacote que mencionou.

Mas se puder, faz um teste aí e posta o código, seria de grande ajuda para outras pessoas.

Só não testa em produção hahahahah

Abraços e bons estudos 🚀

**Comment by dalmiroe** [comment]
Entendi

**Comment by omarceloribeiro** [comment]
Já usei o Pomelo umas 2 vezes no .net para usar o MySQL, acho que esse é o mais usado, não conheço outro.

**Comment by dalmiroe** [comment]
Obrigado Marcelo, usei ele e a migração funcionou
